tests/unit/Develop.spec.js

```
@@ -0,0 +1,29 @@
+import { mount } from '@vue/test-utils'
+import { nextTick } from 'vue'
```

ESLint

[error] 'nextTick' is defined but never used.

(no-unused-vars)